neuro_extras/image_builder.py

```diff
@@ -292,6 +293,16 @@ async def build(
             build_command.append(f"--volume={volume}")
         for env in envs:
             build_command.append(f"--env={env}")
+        for extra_env in KANIKO_EXTRA_ENVS:
+            if extra_env in envs:
```

Comment:
This probably wont handle the case when somebody passes `container=not-docker`, as those are different strings

Reply:
true, will check if there are key duplicates. thank you!